src/utils/fetch-live-event-data.js

```diff
@@ -0,0 +1,28 @@
+const API_KEY = '00411c8fd1b9c3224260259d245dd29890130b94';
+const LIVE_EVENTS_URL = 'https://live.mongodb.com/api/event?status=Live';
+
+// Fetches data from live.mongodb.com events api
+const fetchLiveEventData = async () => {
+    const data = await fetch(LIVE_EVENTS_URL, {
+        headers: {
+            Authorization: `Token ${API_KEY}`,
+            'Content-Type': 'application/json',
+        },
+    });
+    if (data) {
+        const parsedData = await data.json();
+        return parsedData.results.map(e => ({
+            // mongodb.com events api returns relevant fields in `node_type_attributes`
```

Comment:
Perhaps adding a little more context here to document the goal of keeping all events data consistent. Do we have any plans to add typescript at all?

Reply:
No plans yet for TS, but yeah maybe a comment or such would definitely help